src/common/transformations/include/transformations/rt_info/constant_node.hpp

```diff
@@ -0,0 +1,33 @@
+// Copyright (C) 2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "openvino/core/node.hpp"
+#include "openvino/core/runtime_attribute.hpp"
+#include "transformations_visibility.hpp"
+
+namespace ov {
+
+TRANSFORMATIONS_API void mark_as_constant_node(const std::shared_ptr<Node>& node);
+TRANSFORMATIONS_API void mark_as_constant_node(const Output<Node>& node);
+
+TRANSFORMATIONS_API bool is_marked_as_constant_node(const std::shared_ptr<Node>& node);
+TRANSFORMATIONS_API bool is_marked_as_constant_node(const Output<Node>& node);
+
+/**
+ * @ingroup ov_runtime_attr_api
+ * @brief ConstantNode class represents runtime info attribute that marks operation
+ * that are part of constant subgraph.
+ */
+class TRANSFORMATIONS_API ConstantNode : public RuntimeAttribute {
+public:
+    OPENVINO_RTTI("constant_node", "0");
+
+    bool is_copyable() const override {
+        return true;
+    }
```

Comment:
Are you sure that this attribute should be copyable? It seems like this could lead to an unexpected behavior.
For example, we can fuse some op sequence from constant path to some `ov::Node` not from constant path: in this case  `ov::Node` will be unexpectedly marked as constant node.

Reply:
Agree with you, You can check that first state is not to copy this info. But some tests were failed